peep.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Get pip version number
+PIPVER=`pip --version | awk '{print $2}'`
+ARGS=""
+
+echo "peep.sh: Using pip $PIPVER"
+
+# If we're using pip 1.5, 1.6 or 6.0, then toss on the --no-use-wheel
+# argument.
+#
+# Note: If we encounter other versions of pip here that require other
+# things, we can toss them in.
+case $PIPVER in
+    1\.5*|1\.6*|6\.*)
+        echo "peep.sh: Wheel-using pip detected, so passing --no-use-wheel."
+        ARGS="--no-use-wheel"
+        ;;
+esac
+
+# Execute peep with the command line args plus the additional args
+# if any.
+python ./bin/peep-2.1.1.py $@ $ARGS
```

Comment:
You should put `$@` in double quotes, like `"$@"`. It's goofy, an edge case, and probably won't matter, unless some dingus has spaces in a path.

```bash
$ foo() { for i in $@; do echo $i; done }
$ foo "1 2" 3
1
2
3

$ foo() { for i in "$@"; do echo $i; done; }
$ foo "1 2" 3
1 2
3
```